Sources/Views/ReloadableViewUpdateManager.swift

```diff
@@ -18,12 +18,12 @@ protocol ReloadableViewUpdateManagerDelegate: class {
 protocol ReloadableViewUpdateManager {
 
     /// The delegate that this manager performs updates on.
-    weak var delegate: ReloadableViewUpdateManagerDelegate? { get }
+    var delegate: ReloadableViewUpdateManagerDelegate? { get }
 
     /// The operation that this update manager is associated with.
     /// The update manager will stop updating the delegate
     /// if the operation is cancelled or dellocated.
-    weak var operation: Operation? { get }
+    var operation: Operation? { get }
```

Comment:
Why does this need to change?

Reply:
Using `weak` in a protocol declaration will invoke the warning
> 'weak' should not be applied to a property declaration in a protocol and will be disallowed in future versions.

This is the result of proposal https://github.com/apple/swift-evolution/blob/master/proposals/0186-remove-ownership-keyword-support-in-protocols.md